### ISSUE TITLE
EBLattenuation & DoubleSmoothlyBrokenPowerlaw not available

### ISSUE THREAD

**Issue by eneights** [threeML/astromodels#200]
When I try to import the EBLattenuation and DoubleSmoothlyBrokenPowerlaw models, I get the errors
`ImportError: cannot import name 'EBLattenuation' from 'threeML'`
and 
`ImportError: cannot import name 'DoubleSmoothlyBrokenPowerlaw' from 'threeML'`

Running
```
from threeML import *
spectrum = EBLattenuation()
```
gives me the error `NameError: name 'EBLattenuation' is not defined`, and the same thing happens for DoubleSmoothlyBrokenPowerlaw.

I don't have this issue with any other models.

**Comment by henrikef** [comment]
@eneights do you have https://github.com/me-manu/ebltable installed?

**Comment by eneights** [comment]
No, I don't have ebltable installed. So we just need to install that in order to use EBLattenuation?

**Comment by henrikef** [comment]
> No, I don't have ebltable installed. So we just need to install that in order to use EBLattenuation?

Yes, please try installing it and let me know if the error persists.

**Comment by eneights** [comment]
Yes, I'm able to use EBLattenuation after installing ebltable.
